src/shared/lib/localization/en.json

```diff
@@ -53,10 +52,12 @@
     "searchPlaceholder": "Search Address / Txn Hash / Block / Token"
   },
   "blockDetails":{
+    "title": "Block #{height}",
     "parent" : "Parent",
-    "merkle": "Merkle",
-    "invalidated": "Invalidated",
-    "viewChangeProofs": "View Change Proofs"
+    "transactionMerkleRootHash": "Transaction's Merkle Root Hash",
+    "rejectedTransactionMerkleRootHash": "Rejected Transaction's Merkle Root Hash",
+    "invalidatedTransactionHash": "{length} Invalidated Block Hashes",
+    "viewChangeProofs": "{length} View Change Proofs"
```

Comment:
What if `length` is 1? It would be grammatically incorrect:

> 1 Invalidated Block Hashes

Should be:

> 1 Invalidated Block Hash
> 2 Invalidated Block Hashes
> 21 Invalidated Block Hash

Reply:
Acknowledged and committed 👍🏼